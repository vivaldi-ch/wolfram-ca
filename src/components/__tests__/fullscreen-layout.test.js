import React from 'react';
import renderer from 'react-test-renderer';

import FullscreenLayout from '../fullscreen-layout';

describe('Fullscreen Layout', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <FullscreenLayout>
          <h1>This is a test</h1>
        </FullscreenLayout>
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
