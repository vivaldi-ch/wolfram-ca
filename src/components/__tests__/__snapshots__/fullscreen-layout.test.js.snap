// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fullscreen Layout renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": 960,
      "padding": "2em 2em 2em",
    }
  }
>
  <main>
    <h1>
      This is a test
    </h1>
  </main>
</div>
`;
